Terminal.Gui/Views/Menu.cs

```diff
@@ -815,6 +816,7 @@ internal void CloseMenu (bool reopen = false, bool isSubMenu = false)
 					IsMenuOpen = false;
 					PositionCursor ();
 				}
+				IsMenuOpen = false;
```

Comment:
Both branches of the `if` above set `IsMenuOpen = false`. Why is this needed?

Reply:
You're right. My bad :-)